### PR TITLE
First version we tested OK.

### PR DESCRIPTION
**Pull request by RBhupi**
1. runs Mobotix binary with given options, 
2. stops at timeout, 
3. converts the files from RGB to jpeg using FFmpeg
4. renames and extracts timestamp, and removes the files after uploading. 
5. ToDo: plugin.upload_file.

### CODE REVIEW

**Comment by RBhupi** [comment]
- Now runs with pywaggle plugin and uploads the files
- the temp directory is also removed.
- Only one timestep is saved per iteration.
- No errors in this version when run with docker.

File: Dockerfile.mobotix [hunk 1]
```diff
@@ -17,12 +17,18 @@ RUN /usr/bin/make -C /build/eventstreamclient/plugin-client/thermal-raw
...
 RUN apt-get update && apt-get install -y \
-    libboost-program-options1.71.0
+    libboost-program-options1.71.0 \
+    ffmpeg
 
 COPY --from=builder /build/eventstreamclient/plugin-client/thermal-raw/build/thermal-raw /thermal-raw
 
+COPY app /app/
+
 # TODO:
```
Comment (jswantek) [inline Dockerfile.mobotix:32]: remove extra comments

File: app/app.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    
+            exit()
+
+    
+
+    
+
+    
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='''
```
Comment (jswantek) [inline app/app.py:119]: we will need to update these arguments to pull information from the environment variables that will be populated by kubernetes.  See, for example:

https://github.com/waggle-sensor/wes-metrics-agent/blob/main/main.py#L497

**Review by jswantek — CHANGES_REQUESTED** [review]

File: app/app.py (hunk 2 above)
Comment (jswantek) [reply app/app.py:119]: done (i made this change)

File: Dockerfile.mobotix (hunk 1 above)
Comment (jswantek) [reply Dockerfile.mobotix:32]: done (i made this change)

**Comment by jswantek** [comment]
Replaced by https://github.com/waggle-sensor/plugin-mobotix-sampler/pull/4

**Pull request closed without merging**